src/_constants/ActionTypes.js

```diff
@@ -53,3 +53,7 @@ export const UPDATE_VIDEO_LIST = 'UPDATE_VIDEO_LIST';
 
 export const WATCHLIST_FAVOR_ASSET = 'WATCHLIST_FAVOR_ASSET';
 export const WATCHLIST_UNFAVOR_ASSET = 'WATCHLIST_UNFAVOR_ASSET';
+
+export const ADD_TOKEN = 'ADD_TOKEN';
```

Comment:
OK, SO had this haha

http://stackoverflow.com/questions/33637740/should-i-use-one-or-several-action-types-to-represent-this-async-action

they generally recommend to combine, still it depends sometime we might have 2 actions that do the same thing now, but diverge later

generic actions never goes wrong though, it will then depends on caller when somethings need to change

anyway for this case, we should combine them.